src/command.rs

```diff
@@ -154,6 +157,17 @@ pub fn run() -> sc_cli::Result<()> {
 				Ok(())
 			})
 		},
+		Some(Subcommand::Benchmark(cmd)) => {
+			if cfg!(feature = "runtime-benchmarks") {
+				let runner = cli.create_runner(cmd)?;
+
+				runner.sync_run(|config| cmd.run::<kulupu_runtime::Block, service::Executor>(config))
+			} else {
+				println!("Benchmarking wasn't enabled when building the node. \
```

Comment:
Perhaps return an error here?

```suggestion
				Err("Benchmarking wasn't enabled when building the node. \
				     You can enable it with `--features runtime-benchmarks`.to_string())
```